test/e2e/node/security_context.go

```diff
@@ -59,7 +59,7 @@ func scTestPod(hostIPC bool, hostPID bool) *v1.Pod {
 	return pod
 }
 
-var _ = SIGDescribe("Security Context [Feature:SecurityContext]", func() {
+var _ = SIGDescribe("Security Context [Feature:SecurityContext][Feature:RunAsGroup]", func() {
```

Comment:
will adding a feature tag to this exclude these tests if the RunAsGroup feature is not enabled?

Reply:
cc @kubernetes/sig-testing-pr-reviews 
for question about feature tags

Reply:
my main question is how to enable the feature for the RunAsGroup e2e tests and whether  we should remove that tag or not ?

Reply:
first, I don't think you should add this tag to existing tests... we don't want to stop existing tests from running.

it might make more sense to set up a second `SIGDescribe("Security Context [Feature:RunAsGroup]")` block in this file with only RunAsGroup tests inside it. could use help from sig-testing... might be good to ask in slack how these tags are used by the CI jobs

Reply:
looks like lots of other tests do `It("Should do foo [Feature:...]")`... let's just move the `[Feature:RunAsGroup]` tag into your new tests' `It(...)` descriptions

Reply:
> my main question is how to enable the feature for the RunAsGroup e2e tests and whether we should remove that tag or not ?

since the API field is only enabled when the feature gate is enabled, only e2e jobs that enable the feature gate can run these tests successfully. This CI test enables all alpha features, and would run your new e2e tests: https://k8s-testgrid.appspot.com/google-gce#gci-gce-alpha-features

Your new e2e tests would not run in normal merge-queue-blocking CI runs, since those don't enable alpha features